README.md

```diff
@@ -96,21 +96,14 @@ setono_sylius_terms_admin:
 `http://localhost:8000/en_US/terms-conditions`, make sure you don't add
 any terms with slugs like `products`, `taxons`, `login`, etc.
 
-### Step 5: Copy template
 
-```bash
-cp vendor/setono/sylius-terms-plugin/tests/Application/templates/bundles/SyliusShopBundle/Checkout/Complete/_form.html.twig \
-    templates/bundles/SyliusShopBundle/Checkout/Complete/_form.html.twig
-```
-
-# Development
+### Step 5: Add terms to checkout complete form
+Change the view file `templates/bundles/SyliusShopBundle/Checkout/Complete/_form.html.twig`
```

Comment:
Override the [Sylius form](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/Complete/_form.html.twig):
```bash
$ cp vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/Complete/_form.html.twig \
templates/bundles/SyliusShopBundle/Checkout/Complete/_form.html.twig
```

Add this form theme at the top:
```twig
{% form_theme form.terms '@SetonoSyliusTermsPlugin/Shop/Form/termsTheme.html.twig' %}
```

and add this line where you want the terms:

```twig
{{ form_row(form.terms) }}
```

The final file should look something like this:
```twig
{# templates/bundles/SyliusShopBundle/Checkout/Complete/_form.html.twig #}
{% form_theme form.terms '@SetonoSyliusTermsPlugin/Shop/Form/termsTheme.html.twig' %}

{{ form_row(form.notes, {'attr': {'rows': 3}}) }}
{{ form_row(form.terms) }}
```